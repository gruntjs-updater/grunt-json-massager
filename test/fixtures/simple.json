{
  "a": "string",
  "b": 1,
  "c": 1.1,
  "d": 10000000000,
  "e": -1,
  "f": true,
  "g": false,
  "h": null
}