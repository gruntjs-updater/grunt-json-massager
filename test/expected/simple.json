{
  "a": "string",
  "b": 1,
  "c": 1.1,
  "d": 1e10,
  "e": -1,
  "f": true,
  "g": false,
  "h": null
}
